### DISCUSSION TITLE
Safari 15 Poor UI

### DISCUSSION THREAD

**Discussion by andrew-edgar** [concourse/concourse#7587] [General]
Has anybody else seen that the latest Safari 15 does not work well with concourse UI.  I see lots of problems with the header becoming very large and icons overlapping etc when navigating around the concourse UI. 

We have v7.4.0 of concourse running.

**Comment by taylorsilva** [comment]
Yup that looks bad
<img width="1510" alt="image" src="https://user-images.githubusercontent.com/4583837/134570183-b66907ea-639c-48fe-b1a4-5928785ff697.png">

**Reply by taylorsilva** [reply]
<img width="1507" alt="image" src="https://user-images.githubusercontent.com/4583837/134570298-f355982f-0e40-4d26-a4b2-46de4b828a63.png">

**Reply by taylorsilva** [reply]
weird, gets worse as I navigate more. If i refresh any page I'm on it renders correctly initially. Clicking around to other pages slowly makes things worse.

**Reply by andrew-edgar** [reply]
yeah not a great experience.  @taylorsilva are you investigating or opening an issue?

**Reply by taylorsilva** [reply]
Made an issue for it. Haven't investigated.

**Reply by taylorsilva** [reply]
I started looking into this and I think it's actually a Safari bug. No errors print to the JS console. I found refreshing the page after rendering issues occur fixes the problem for the life of the tab. Updated the issue with this new info. Submitted a bug report to Safari. Let's hope it hasn't land in a blackhole somewhere 😬

**Reply by taylorsilva** [reply]
https://github.com/concourse/concourse/issues/7605
